megamek/data/forcegenerator/factions.xml

```diff
@@ -454,12 +445,12 @@
 	</faction>
 	<faction key='RR' name='Republic Remnant' minor='false' clan='false' periphery='false'>
 		<years>3135-</years>
-		<ratingLevels>Triarii Protectors,Principes Guards,Hastati Sentinels,Stone&apos;s Brigade</ratingLevels>
+		<ratingLevels>Triarii Protectors,Principes Guards,Hastati Sentinels,Stone's Brigade</ratingLevels>
 		<parentFaction>ROS</parentFaction>
 	</faction>
 	<faction key='ROS' name='Republic of the Sphere' minor='false' clan='false' periphery='false'>
 		<years>3081-</years>
-		<ratingLevels>Triarii Protectors,Principes Guards,Hastati Sentinels,Stone&apos;s Brigade</ratingLevels>
+		<ratingLevels>Triarii Protectors,Principes Guards,Hastati Sentinels,Stone's Brigade</ratingLevels>
```

Comment:
Why the change? This really should be `&apos;` or wrapped in a CDATA.

Reply:
Added a function call to escape these.